entries/n/n8n.io.json

```
@@ -0,0 +1,16 @@
+{
+  "n8n": {
+    "domain": "n8n.io",
+    "additional-domains": [
+      "n8n.cloud"
+    ],
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.n8n.io/user-management/two-factor-auth/",
+    "categories": [
+      "cloud",
+      "other"
```

```suggestion
      "developer"
```

I think the `developer` category is better, especially considering that [Zapier is categorized as such](https://github.com/2factorauth/twofactorauth/blob/master/entries/z/zapier.com.json).